source/_integrations/escea.markdown

```diff
@@ -0,0 +1,48 @@
+---
+title: Escea
+description: Instructions on how to integrate Escea fireplaces with Home Assistant.
+ha_category:
+  - Climate
+ha_release: 2022.9
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@lazdavila'
+ha_domain: escea
+ha_homekit: true
+ha_platforms:
+  - climate
+---
+
+The `Escea` integration allows monitoring and control of local [Escea](https://escea.com/) fireplaces. These fireplaces are manufactured in New Zealand and sold throughout Australia and New Zealand.
+
+## Supported hardware
+
+Support all current Escea fireplaces with the Smart Heat feature, including: DF; DL; DS; and, DX series.
+Though these fireplaces are supported, they can only be integrated if they were connected via Ethernet when installed (i.e. these fireplaces do not support Wifi).
+
+{% include integrations/config_flow.md %}
+
+
+## Network settings
+
+The Escea system uses UDP over the local network to find and communicate with Escea devices. For this to work properly, UDP port  3300 must be available for broadcasting and messaging fireplaces, and similarly port UDP port 3300 must be available to listen for responses. The integration currently listens on `0.0.0.0` and broadcasts to all broadcast IPv4 local addresses, which is not configurable.
```

Comment:
```suggestion
The Escea system uses UDP over the local network to find and communicate with Escea devices. For this to work properly, UDP port 3300 must be available for broadcasting and messaging fireplaces, and similarly, UDP port 3300 must be available to listen for responses. The integration currently listens on `0.0.0.0` and broadcasts to all broadcast IPv4 local addresses, which is not configurable.
```

Reply:
Shouldn't it be listening to the interfaces configured in the Home ASsistant network manager?

Reply:
Hi Franck, thanks for the review. I don't know about configuring the Network Manager. In my local setup I did not change anything there. I just had a look and there are no network rules allowing or blocking these UDP ports. The integration seems to work fine without any configuration on my local system. If I need to do something, could you please let me know where to look? Alternatively, if you think I should get rid of the comments about what UDP ports it uses (so as not to confuse people) then please let me know.